src/app/auth/auth.interceptor.ts

```diff
@@ -12,14 +12,15 @@ export class AuthInterceptor implements HttpInterceptor {
     intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
         if(localStorage.getItem('token')!=null) {
             const clonedReq = req.clone({
-                headers : req.headers.set('Authorization','Bearer '+localStorage.getItem('token'))
+                headers : req.headers.set('Authorization','Bearer '+localStorage.getItem('accessToken'))
             });
             return next.handle(clonedReq).pipe(
                 tap(
                     succ => {},
                     err => {
                         if(err.status == 401) {
-                            localStorage.removeItem('token');
+                            localStorage.removeItem('AccessToken');
```

Comment:
Keep name consistent. AccessToken or accessToken either of them.

Reply:
changes done